<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>185</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Server Setting</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>293</width>
      <height>159</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLabel" name="label_ip">
               <property name="text">
                <string>&amp;IP address</string>
               </property>
               <property name="buddy">
                <cstring>lineEdit_ip</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_port">
               <property name="text">
                <string>&amp;Port</string>
               </property>
               <property name="buddy">
                <cstring>lineEdit_port</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_users">
               <property name="text">
                <string>&amp;Max users count</string>
               </property>
               <property name="buddy">
                <cstring>lineEdit_users</cstring>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLineEdit" name="lineEdit_ip"/>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_port"/>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_users"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_info">
         <property name="text">
          <string>Input setting</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_cancel">
         <property name="text">
          <string>&amp;Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_default">
         <property name="text">
          <string>&amp;Default</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_accept">
         <property name="text">
          <string>&amp;Accept</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEdit_ip</tabstop>
  <tabstop>lineEdit_port</tabstop>
  <tabstop>lineEdit_users</tabstop>
  <tabstop>pushButton_cancel</tabstop>
  <tabstop>pushButton_default</tabstop>
  <tabstop>pushButton_accept</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_cancel</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>22</x>
     <y>166</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
